fastapi_contrib/common/responses.py

```diff
@@ -1,10 +1,10 @@
 import typing
 import ujson
 
-from starlette.responses import UJSONResponse as BaseUJSONResponse
+from starlette.responses import JSONResponse
 
 
-class UJSONResponse(BaseUJSONResponse):
+class UJSONResponse(JSONResponse):
```

Comment:
Well, you can't just inherit from different json serializer and expect for people to know that ujson response is now actually just plain json response.

Since there is no UJSONResponse in starlette in the latest version, we need to implement it ourselves. Luckily, it is quite easy to do as per https://www.starlette.io/responses/#custom-json-serialization.

Reply:
Yes, and UJSONResponse does implement that, as it shown [here](https://github.com/encode/starlette/pull/1047#issue-480711499), so I don't see the issue here to be honest